MobileFilter/sections/specific_web.txt

```diff
@@ -51,6 +51,9 @@ analdin.com##.banner-top
 pluggedin.ru##a[target="_blank"][href^="https://ads.adfox.ru"] > img
 community.spiceworks.com##.ad-content-slot
 community.spiceworks.com##.ad-leaderboard-slot
+up.ymgt.xyz,google.de,www-bento-de.cdn.ampproject.org##.ads
+up.ymgt.xyz#$#.play-wrap > .game { height: 100%!important; }
```

Comment:
to `general_extensions.txt`

Reply:
@Alex-302,
moved #$# rule to `general_extensions.txt`